test/sourcemaps/index.js

```diff
@@ -88,6 +88,10 @@ test('fails', function (t) {
 
 function onError (t, assert) {
   agent._transport = {
+    flush () {},
+    sendTransaction () {},
+    sendSpan () {},
+    sendMetricSet () {},
```

Comment:
@Qard as far as I can see this shouldn't be affected by the change in this PR. If I comment it out and run the test, it parses fine. Do you remember it was added it?

Reply:
If you comment out `sendMetricSet` it may fail intermittently because the test does not set `metricsInterval: 0`. We should _probably_ actually be setting that too, but I wanted to have those functions in there just for _completeness_. Something could change in the future which results in those functions getting _called_ in there. At the least, `flush(...)` has a high probably of being called before it completes.